src/compositemark/errorband.ts

```diff
@@ -43,6 +45,11 @@ export interface ErrorBandConfig extends ErrorBandPartsMixins {
    * __Default value:__ `"stderr"`.
    */
   extent?: ErrorBarExtent;
+
+  /**
+   * The interpolate options for error band parts.
```

Comment:
Let's copy docs from normal `interpolate` so it includes all the interpolation property and add this property the errorband.md. :)